src/main/java/frc/robot/subsystems/ball/shooter/Shooter.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot.subsystems.ball.shooter;
+
+public interface Shooter {
+    void runShooter();
+    void stopShooter();
+    void invertPosition();
```

Comment:
This is a confusing method. Is the "position" to do with a shooter hood or something? If so, it probably makes more sense to have the interface define something like `setPosition(ShooterPosition position)`/`ShooterPosition getPosition()` or `setHoodRaised(boolean raised)`/`boolean isHoodRaised()` and let the parent subsystem handle the decision of what "inverting" means.

Reply:
Yeah, as far as I know the shooter has a hood. That might have changed so I'll verify that.